### DISCUSSION TITLE
Why doesn't vendure have a discord?

### DISCUSSION THREAD

**Discussion by JustDenP** [vendurehq/vendure#2011] [General]
As far as I understand, the vendure community has no discord server and only slack.

I have nothing against it, but I think the lack of discord is a pretty big omission for community development in general for several reasons:

A few years ago I also perceived it only as something "around gaming", but in recent years its audience has grown so much, and the plugins and usability have made it very convenient for any kind of community. So it's not surprising that even many big companies use it to connect with their audience. You can also look at the example of vendure's "competitors". (Even NestJs itself has a discord server, where you can find a lot of useful information from the community), so I think we should keep up with them.

The second reason is that in many countries slack is not a common service (it can be used only in large IT companies, and then not always, because there are analogues), and discord is used by almost everyone. I think just create a server and a few thematic channels will take from half an hour, but so we can at least see whether it will be relevant to people.

**Comment by michaelbromley** [comment]
Hi Denis,

Discord has improved a lot in recent years - back when I was choosing a platform for the Vendure community I really disliked Discord. It's lack of threads made the channels like a firehose of context-less messages. So back then, Slack was the clear winner.

Since then, they got threads (though I still much prefer the Slack implementation) and even a forum-like interface. So now if I were choosing a platform today, I would seriously consider Discord over Slack. 

I still prefer the experience of using Slack, but the limitations of the free tier are annoying - all messages older than 90 days are lost. And there is zero chance of us ever moving to a paid plan with almost 2,000 members!

Right now I think the main issue would be having to migrate our existing community over to a new platform. It's not out of the question, but I'm not convinced that Discord is so clearly superior to Slack to make me want to do it right now. Can you tell me more about how Slack "can be used only in large IT companies" in some countries? I've never heard about that before.
